includes/migration-guide/retargeting/wpf/accessibility-improvements-wpf.md

```diff
@@ -2,7 +2,7 @@
ntrols.CheckBox> and <xref:System.Windows.Controls.RadioButton> controls when they are selected is now easier to see than in previous .NET Framework versions.</li><li>The border of a disabled <xref:System.Windows.Controls.ComboBox> is now the same color as the disabled text. In previous versions of the .NET Framework, it was not.</li><li>Disabled and focused buttons now use the correct theme color. In previous versions of the .NET Framework, they did not.</li><li>The dropdown button is now visible when a <xref:System.Windows.Controls.ComboBox> control&#39;s style is set to <xref:System.Windows.Controls.ToolBar.ComboBoxStyleKey?displayProperty=nameWithType>. In previous versions of the .NET Framework, it was not.</li><li>The sort indicator arrow in a <xref:System.Windows.Controls.DataGrid> control now uses theme colors. In previous versions of the .NET Framework, it did not.</li><li>The default hyperlink style now changes to the correct theme color on mouse over. In previous versions of the .NET Framework, it did not.</li><li>The Keyboard focus on radio buttons is now visible. In previous versions of the .NET Framework, it was not.</li><li>The <xref:System.Windows.Controls.DataGrid> control&#39;s checkbox column now uses the expected colors for keyboard focus feedback. In previous versions of the .NET Framework, it did not.</li><li>the Keyboard focus visuals are now visible on <xref:System.Windows.Controls.ComboBox> and <xref:System.Windows.Controls.ListBox>. In previous versions of the .NET Framework, it was not.</li></ul><strong>Screen reader interaction improvements</strong><ul><li><xref:System.Windows.Controls.Expander> controls are now correctly announced as groups (expand/collapse) by screen readers.</li><li><xref:System.Windows.Controls.DataGridCell> controls are now correctly announced as data grid cell (localized) by screen readers.</li><li>Screen readers will now announce the name of an editable <xref:System.Windows.Controls.ComboBox>.</li><li><xref:System.Windows.Controls.PasswordBox> controls are no longer announced as �no item in view� by screen readers.</li></ul><strong>LiveRegion support</strong>Screen readers such as Narrator help people know the UI contents of an application, usually by describing something about the UI that�s currently focused, because that is probably the element of most interest to the user. However, if a UI element changes somewhere in the screen and it does not have the focus, the user may not be informed and miss important information. LiveRegions are meant to solve this problem. A developer can use them to inform the screen reader or any other [UI Automation][UI Automation Overview](~/docs/framework/ui-automation/ui-automation-overview.md) client that an important change has been made to a UI element. The screen reader can then decide how and when to inform the user of this change. The LiveSetting property also lets the screen reader know how important it is to inform the user of the change made to the UI.|
```

Comment:
I caught some unprintable quote characters in this paragraph. Do we want to fix them in this PR?

Reply:
Thanks for noticing these, @BillWagner. I've fixed them in this PR.